src/io_hid_manager.rs

```diff
@@ -0,0 +1,24 @@
+// exports from <IOKit/hid/IOHIDManager.h>
+
+use libc::c_void;
+use cf::{CFAllocatorRef, CFDictionaryRef};
+use io_return::IOReturn;
+use io_hid_keys::IOHIDManagerOptions;
+use io_hid_base::{IOHIDDeviceCallback, IOHIDReportCallback};
+
+#[doc(hidden)]
+#[repr(C)]
+pub struct __IOHIDManager {
+    __private: c_void,
+}
+pub type IOHIDManagerRef = *mut __IOHIDManager;
+
+extern "C" {
+    pub fn IOHIDManagerCreate(allocator: CFAllocatorRef, options: IOHIDManagerOptions) -> IOHIDManagerRef;
```

Comment:
`options` is defined as `IOOptionBits` in the header file I'm looking at:

```c
CF_EXPORT 
IOHIDManagerRef IOHIDManagerCreate(     
                                CFAllocatorRef                  allocator,
                                IOOptionBits                    options)
AVAILABLE_MAC_OS_X_VERSION_10_5_AND_LATER;
```

`IOOptionBits` is a typedef for `u32`, while `IOHIDManagerOptions` is a typedef for `c_int`, which isn't guaranteed to be the same size. Since Rust is stricter than C about integer conversions, I'm guessing that you changed the type here to prevent needing to type `as IOOptionBits` over and over. If this is the only place that `IOHIDManagerOptions` is intended to be used, I think it would make sense to change the target of the `IOHIDManagerOptions` typedef to `IOOptionBits`, since the C compiler would make the same conversion automatically.